website/docs/r/stack_hci_deployment_setting.html.markdown

```diff
@@ -0,0 +1,585 @@
+
+* `version` - (Required) The deployment template version, possible value is `10.0.0.0`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `adapter_property_override` block supports the following:
+
+* `jumbo_packet` - (Optional) The jumbo frame size of the adapter. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `network_direct` - (Optional) The network direct of the adapter. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `network_direct_technology` - (Optional) The network direct technology of the adapter. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `cluster` block supports the following:
+
+* `azure_service_endpoint` - (Required) For Azure blob service endpoint type, select either Default or Custom domain. If you selected Custom domain, enter the domain for the blob service in this format `core.windows.net`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `cloud_account_name` - (Required) Specify the Azure Storage account name for cloud witness for your Azure Stack HCI cluster. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `name` - (Required) The cluster name provided when preparing Active Directory. It must be 3-15 characters long and contain only letters, numbers and hyphens. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `witness_path` - (Required) Specify the fileshare path for the local witness for your Azure Stack HCI cluster. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `witness_type` - (Required) Use a cloud witness if you have internet access and if you use an Azure Storage account to provide a vote on cluster quorum. A cloud witness uses Azure Blob Storage to read or write a blob file and then uses it to arbitrate in split-brain resolution. Possible values are `Cloud`, `FileShare`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `host_network` block supports the following:
+
+* `intent` - (Required) One or more `intent` blocks as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `storage_network` - (Required) One or more `storage_network` blocks as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `storage_auto_ip_enabled` - (Optional) Whether allows users to specify IPs and Mask for Storage NICs when Network ATC is not assigning the IPs for storage automatically. Optional parameter required only for 3 Nodes Switchless deployments. Possible values are `true` and `false`. Defaults to `true`. Changing this forces a new Stack HCI Deployment Setting to be created.
```

Comment:
what does it mean for `Optional parameter required only for 3 Nodes Switchless deployments`?